### PR TITLE
Accessibility Section Revamp

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: accessibility/compliance.md [hunk 1]
```diff
@@ -5,35 +5,140 @@ description: Compliance with the accessibility standards and requirements in the
...
+
+The information is subject to the following considerations:
+
+* All components implement the required WAI-ARIA attributes without the need for any extra configuration. Some components may provide parameters that render additional optional WAI-ARIA attributes, for example, `aria-label` or `aria-describedby`.
+* The compliance levels are achievable with the [*Default Ocean Blue A11y* theme swatch]({%slug accessibility-overview%}#color-contrast).
+* The accessibility and compliance of some components may depend on the enabled features. In such cases, the compliance level below is based on the default component configuration.
+* Component templates introduce custom markup that may not be accessible. Test any modifications to ensure the web content still meets the desired level of accessibility compliance. Be mindful of components that work with user input such as images, text, or HTML content.
+* Due to the complexity of some components, there are scenarios that are not covered by the WAI-ARIA specification. In those cases, we benefit from the know-how of the dedicated accessibility professionals at Progress Software and feedback from accessibility-minded users.
```
Comment (yordan-mitev) [inline accessibility/compliance.md:32]: What does this mean for the Telerik clients? How does it affect them?

File: accessibility/overview.md [hunk 2]
```diff
@@ -5,26 +5,100 @@ description: Accessibility features in the Telerik UI for Blazor suite.
...
+
+Most components represent a single tab stop. Once users reach and focus a component, they can leave it with a single `Tab` key press. If the component is more complex, users can walk through its inner elements with the arrow keys, for example, Grid cells, Menu items, Toolbar buttons. Some complex components can accommodate multiple other components. For example, the Grid can host a Toolbar and a Pager. In this case, tab to move the focus from one nested component to another.
+
+### Types of Keyboard Support
+
+The Telerik UI for Blazor components may provide enhanced, standard, or no keyboard support. See the [compliance table]({%slug accessibility-compliance%}#compliance-table) for component-specific information.
+
+* *Standard keyboard support* implies similar keyboard navigation capabilities as standard HTML elements. For example, the Button components support `Enter` and `Space` for clicking them. All components with standard keyboard support are reachable through the `Tab` key and provide focus styles.
```
Comment (yordan-mitev) [inline accessibility/overview.md:64]:
```suggestion
* *Standard keyboard support* implies similar keyboard navigation capabilities as standard HTML elements. For example, the Button components support `Enter` and `Space` for triggering clicks. All components with standard keyboard support are reachable through the `Tab` key and provide focus styles.
```

File: components/autocomplete/accessibility/wai-aria-support.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ position: 50
...
 
 
-The AutoComplete is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2 AA](https://www.w3.org/TR/WCAG22/) standards and [Section 508](https://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation]({%slug accessibility-keyboard-navigation%}) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
+The AutoComplete is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2 AA](https://www.w3.org/TR/WCAG22/) standards and [Section 508](https://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```
Comment (yordan-mitev) [inline components/autocomplete/accessibility/wai-aria-support.md:19]: - The slug for the keyboard navigation link is missing.
- This comment applies to all articles of this type.

File: accessibility/overview.md [hunk 4]
```diff
@@ -5,26 +5,100 @@ description: Accessibility features in the Telerik UI for Blazor suite.
...
+previous_url: /accessibility/keyboard-navigation,/accessibility/accessibility-swatch
 ---
 
 # Blazor Accessibility Overview
 
-Websites and applications are accessible when they provide full control over their features by enabling users with disabilities to access their content through assistive technologies or keyboard navigation.
+Web applications are accessible when they provide control over their features to users with physical or situational disabilities. These disabilities can include various categories, for example, user who are:
 
-Accessibility consists of several aspects:
+* Unable to perceive content visually or auditory;
```
Comment (yordan-mitev) [inline accessibility/overview.md:16]:
```suggestion
* Unable to perceive content visually or auditory.
```

File: accessibility/compliance.md (hunk 1 above)
Comment (dimodi) [reply accessibility/compliance.md]: These are mostly disclaimers, which imply that the accessibility compliance in the customer's app may not match what we claim in the table.

File: components/autocomplete/accessibility/wai-aria-support.md (hunk 3 above)
Comment (dimodi) [reply components/autocomplete/accessibility/wai-aria-support.md:19]: These are now fixed

File: accessibility/overview.md [hunk 5]
```diff
@@ -5,26 +5,113 @@ description: Accessibility features in the Telerik UI for Blazor suite.
...
+
+* [Telerik UI for Blazor version 4.0.1](https://www.telerik.com/support/whats-new/blazor-ui/release-history/ui-for-blazor-4-0-1)
+* [Themes version 6.0.3](https://github.com/telerik/kendo-themes/releases/tag/v6.0.3). Check section [Theme Version Compatibility]({%slug themes-overview%}#compatibility-and-maintenance) on how to align Telerik UI for Blazor versions with theme versions.
+
+## Development Practices
+
+When implementing a Telerik Blazor component, the team:
+
+* Follows the WCAG standard and WAI-ARIA specification to lay the right foundation for the component accessibility.
+* Implements automated unit tests guarantee accessible and semantically correct rendering.
```
Comment (yordan-mitev) [inline accessibility/overview.md:90]:
```suggestion
* Implements automated unit tests to guarantee accessible and semantically correct rendering.
```

File: accessibility/overview.md [hunk 6]
```diff
@@ -5,26 +5,113 @@ description: Accessibility features in the Telerik UI for Blazor suite.
...
+* Localizes messages for labels, titles, and other elements.
+* Manually tests the component to with regard to its keyboard navigation and usage with screen readers.
```
Comment (yordan-mitev) [inline accessibility/overview.md:92]:
```suggestion
* Manually tests the component with regard to its keyboard navigation and usage with screen readers.
```

**Review by xristianstefanov — APPROVED** [review]

**Pull request merged**